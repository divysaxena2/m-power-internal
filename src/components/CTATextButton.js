import React, { Component } from 'react';
import getTheme from '../../native-base-theme/components'
import material from '../../native-base-theme/variables/material';
import { StyleSheet, View } from 'react-native';
import { StyleProvider, Button, Text } from 'native-base';


type Props = {};
export default class CTATextButton extends Component<Props> {

    render() {
        return (
            <StyleProvider style={getTheme(material)}>
                <Text style={styles.buttonText}>View More</Text>
            </StyleProvider>)
    }
}
const styles = StyleSheet.create({

    buttonText: {
        textDecorationLine: 'underline',
        height: 18,
        paddingLeft: 39,
        opacity: 0.54,
        color: '#F7B500',
        //font- family: "Roboto Black";
        fontSize: 13,
        letterSpacing: 0.62,
        lineHeight: 18,
        textAlign: 'center'
    }
});