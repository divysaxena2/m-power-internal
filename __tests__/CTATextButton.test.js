import 'react-native';
import React from 'react';


import renderer from 'react-test-renderer';
import CTATextButton from '../src/components/CTATextButton';

it('renders correctly', () => {
    const tree = renderer.create(
        <CTATextButton />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});