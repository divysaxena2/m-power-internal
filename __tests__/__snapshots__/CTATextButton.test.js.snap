// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
        "fontFamily": "Roboto",
        "fontSize": 16,
      },
      Object {
        "color": "#F7B500",
        "fontSize": 13,
        "height": 18,
        "letterSpacing": 0.62,
        "lineHeight": 18,
        "opacity": 0.54,
        "paddingLeft": 39,
        "textAlign": "center",
        "textDecorationLine": "underline",
      },
    ]
  }
  uppercase={false}
>
  View More
</Text>
`;
